src/transformers/models/t5/modeling_t5.py

```diff
@@ -700,7 +700,8 @@ def forward(
         else:
             outputs = outputs + attention_outputs
 
-        return outputs  # hidden-states, present_key_value_states, (self-attention weights), (self-attention position bias), (cross-attention weights), (cross-attention position bias)
+        outputs = outputs + (present_key_value_state,) + attention_outputs
```

Comment:
```suggestion
```

This was not intentional no?

Reply:
The rest looks good to me

Reply:
@patrickvonplaten Yes, that was an accident. Thank you!